store/storage/pebbledb/db_test.go

```diff
@@ -12,12 +12,14 @@ import (
 func TestStorageTestSuite(t *testing.T) {
 	s := &storage.StorageTestSuite{
 		NewDB: func(dir string) (store.VersionedDatabase, error) {
-			return New(dir)
+			db, err := New(dir)
+			if err == nil && db != nil {
+				db.SetSync(false)
```

Comment:
in most environments this would be true, is there a reason to setting this to false for testing?

Reply:
Yes, just to make the unit tests faster. Not a strong reason, but it helps speed up CI testing time.

Reply:
amazing!! could you add a short comment saying this. Would help for future readers

Reply:
gotcha